testsuite/tests/singlecluster/authorino/identity/subject_access_review/conftest.py

```diff
@@ -0,0 +1,49 @@
+"""Conftest for SubjectAccessReview related tests."""
+
+import pytest
+
+from testsuite.httpx.auth import HeaderApiKeyAuth
+from testsuite.kubernetes.cluster_role import ClusterRole, ClusterRoleBinding
+
+
+@pytest.fixture(scope="module")
+def create_cluster_role(request, cluster, blame, module_label):
+    """Creates and returns a ClusterRole"""
+
+    def _create_cluster_role(rules):
```

Comment:
You don't need to return the function here. Why don't you create a role in this fixture directly? The same applies for `create_service_account` but it is for another PR.

Reply:
true, thank you.